### ISSUE TITLE
[Insomnia Designer] Themes of Core Default's contrast ratio too low

### ISSUE THREAD

**Issue by WillGoGithub** [getinsomnia/insomnia#2280]
**Describe the bug**
In the **Preview mode** there are two problems of context in the info.description[markdown.list].
1. The contrast ratio is too low.
2. Missing decoration.

**To Reproduce**
1. Go to 'Application > Preferences > Themes'
2. Click on 'Core Default'
3. Create blank document
4. Add the following code
```
openapi: 3.0.1
info:
  title: Swagger Petstore
  description: |-
    This is a sample server Petstore server.  You can find out more about     Swagger
    at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).      For
    this sample, you can use the api key `special-key` to test the authorization     filters.'

    + list A
    + list B
```

**Expected behavior**

1. High contrast ratio.
2. Have decoration like below.
- list A
- list B

**Screenshots**
![image](https://user-images.githubusercontent.com/45837931/84339755-b6b43780-abd1-11ea-8c19-e840ef73ca26.png)

**Desktop:**
 - OS: Windows 10
 - App Version: 2020.2.2
